student/src/main/java/tds/student/web/controls/dummy/GlobalJavascript.java

```diff
@@ -145,10 +149,14 @@ public void writeJavascript (ResponseWriter output) throws IOException, ReturnSt
 
       // ClientSide
       contextList.addAll (Arrays.asList (jsArray));
-    } else if (StringUtils.equals (contextName, "ReviewShell")) {
+    } else if (StringUtils.equals (contextName, CONTEXT_REVIEW_SHELL)) {
       String[] morePages = { "Student.Master", "TestReview.aspx", "TestResults.aspx" };
       // ServerSide
       contextList.addAll (Arrays.asList (morePages));
+
+    } else if (StringUtils.equals(contextName, CONTEXT_DIAGNOSTIC_SHELL)) {
+      String[] pagesArray = { "Default.aspx", "Diagnostics.aspx", "SoundCheck.aspx", "TTSCheck.aspx" };
+      contextList.addAll (Arrays.asList (pagesArray));
```

Comment:
This is the big change that adds translations needed by the diagnostic shell to the global javascript initialization tag so the page is initialized with:
```
TDS.Config.messages = [many translation key/value pairs]
```